### PR TITLE
[CommandPropertyToAttributeRector] Also get other attribute values (fix #305)

### PR DESCRIPTION
**Pull request by JohJohan**

### CODE REVIEW

**Comment by johanadivare** [comment]
The [unit](https://github.com/rectorphp/rector-symfony/actions/runs/3637517187/jobs/6138616339) test fails because argument `key` are removed, should we add these keys by default what is the best practice for this?

[phpstan](https://github.com/rectorphp/rector-symfony/actions/runs/3637517227/jobs/6138616959)
 fails because ```  src/Rector/Class_/CommandPropertyToAttributeRector.php:35
 ------------------------------------------------------------------------
  - '#Class cognitive complexity is 63, keep it under 50#'``` any idea how i can make this less complex?

File: tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (johanadivare) [inline tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc:1]: Should fixture file names be named `like_this.php.inc` or `likeThis.php.inc`?

Comment (samsonasik) [reply tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc:1]: `like_this.php.inc`

**Comment by OskarStark** [comment]
Thank you very much!

File: tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc (hunk 1 above)
Comment (johanadivare) [reply tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc]: I updated the test fixture file names for CommandPropertyToAttributeRector

**Comment by OskarStark** [comment]
Any chances to get this shipped in the next release?

**Comment by TomasVotruba** [comment]
> The [unit](https://github.com/rectorphp/rector-symfony/actions/runs/3637517187/jobs/6138616339) test fails because argument key are removed,

The keys should be kept. The code should not be changed at all and `refactor()` should return `null`.

> any idea how i can make this less complex?

This means a service should be extracted

**Comment by johanadivare** [comment]
> Any chances to get this shipped in the next release?

Not sure when i'm able to finish this up, i only need to make the class less complex still

**Comment by samsonasik** [comment]
You can create another service, and inject into construct method, then use the service

**Comment by johanadivare** [comment]
> You can create another service, and inject into construct method, then use the service

I created a CommandHelper class to move some functions let me know if that is okay

**Review by samsonasik — APPROVED** [review]
Looks good

File: src/Helper/CommandHelper.php [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+use Rector\NodeNameResolver\NodeNameResolver;
+use Rector\NodeRemoval\NodeRemover;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Rector\NodeTypeResolver\NodeTypeResolver;
+use Rector\Php80\NodeAnalyzer\PhpAttributeAnalyzer;
+use Rector\PhpDocParser\NodeTraverser\SimpleCallableNodeTraverser;
+
+/**
+ * @see \Rector\Symfony\Tests\Rector\ClassMethod\TemplateAnnotationToThisRenderRector\TemplateAnnotationToThisRenderRectorTest
+ */
```
Comment (samsonasik) [inline src/Helper/CommandHelper.php:27]: The `@see` seems not needed as pointed to different test

Comment (johanadivare) [reply src/Helper/CommandHelper.php:27]: Good find i copy paste it from other class and forgot this.

**Comment by samsonasik** [comment]
Let's merge it so we have faster feedback 👍

**Comment by samsonasik** [comment]
Thank you @JohJohan

**Pull request merged**